wolfcrypt/src/rsa.c

```diff
@@ -881,18 +885,22 @@ static int RsaMGF1(enum wc_HashType hType, byte* seed, word32 seedSz,
         /* find largest amount of memory needed which will be the max of
          * hLen and (seedSz + 4) since tmp is used to store the hash digest */
         tmpSz = ((seedSz + 4) > (word32)hLen)? seedSz + 4: (word32)hLen;
```

Comment:
Where are you checking `tmpSz` isn't greater than `RSA_MAX_SIZE/8` (`sizeof(tmp)`)?

Reply:
@dgarske, Added `tmpSz` check for clarity. `seedSz` is bounded in the caller function and it can have a max value of WC_MAX_DIGEST_SIZE.